ui/pages/_app.tsx

```diff
@@ -1,4 +1,4 @@
-import { useState, useEffect } from "react";
```

Comment:
A superadmin doesn't seem to be able to create a new bucket

Reply:
This is intentional, and the solution is to first invite yourself to the round. That way it will be less confusing for the group admins and participants when a new bucket appears.